### ISSUE TITLE
Plot Accuracy and losss curve for train and validation

### ISSUE THREAD

**Issue by elaheyenaz65** [kenshohara/3D-ResNets-PyTorch#208]
@kenshohara 
Hi
Thank you for sharing your code.
I just run the code for UCF101 dataset. Is there any function that plots the results over epoch for train and validation set?

**Comment by Deepanshu-beep** [comment]
> 
> 
> @kenshohara
> Hi
> Thank you for sharing your code.
> I just run the code for UCF101 dataset. Is there any function that plots the results over epoch for train and validation set?

Hey, did you find out any solution for it?
